qiskit_ibm_runtime/session.py

```diff
@@ -179,10 +176,19 @@ def run(
 
         return job
 
-    def close(self) -> None:
-        """Close the session."""
+    def cancel(self) -> None:
+        """Cancel all jobs in a session."""
         self._active = False
         if self._session_id:
+            self._service._api_client.cancel_session(self._session_id)
+
+    def close(self) -> None:
+        """Update the session so new jobs will not be accepted, but existing
+        queued or running jobs will run to completion. The session will be closed when there
+        are no more jobs to run."""
+        backend = self._service.backend(self._backend)
+        # There is a 500 internal error on IQP if the backend is a simulator
+        if not backend.simulator and self._session_id:
```

Comment:
I'll have to test this again and ask the IQP team, I don't this should error when using simulators